nbconvert/exporters/pdf.py

```diff
@@ -74,11 +75,16 @@ def run_command(self, command_list, filename, count, log_function):
         
         times = 'time' if count == 1 else 'times'
         self.log.info("Running %s %i %s: %s", command_list[0], count, times, command)
+        
+        shell = (sys.platform == 'win32')
+        if shell:
+            command = subprocess.list2cmdline(command)
+        env = os.environ.copy()
+        env['TEXINPUTS'] = ':'.join([self.texinputs, env.get('TEXINPUTS', '')])
```

Comment:
This gives me `i:\\notebook:` instead of `i:\\notebook` as `TEXINPUTS` env. The trailing `:` makes the compilation fail.